src/loader.js

```diff
@@ -130,7 +130,13 @@ export function pitch(request) {
     }
     let resultSource = `// extracted by ${pluginName}`;
     if (locals && typeof resultSource !== 'undefined') {
-      resultSource += `\nmodule.exports = ${JSON.stringify(locals)};`;
+      Object.keys(locals).forEach((exportName) => {
+        const className = locals[exportName];
+
+        resultSource += `\nexport const ${exportName} = '${className}';`;
```

Comment:
Will this work with class names that have hyphens in them? Especially if `css-loader` isn't specifically configured to convert the locals into camelCase or something.